### PR TITLE
Cut 0.32.0 release

### PR DESCRIPTION
**Pull request by LiliC**
Starting the 0.32.0 release. Changelog entries taken from [here](https://github.com/coreos/prometheus-operator/compare/release-0.31...master ), think that should be all?

cc @s-urbaniak @paulfantom @brancz  

cc @metalmatze The generation worked finally 🎉  :D

### CODE REVIEW

**Comment by s-urbaniak** [comment]
lgtm! :tada:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+## 0.32.0 / 2019-08-30
+
+* [FEATURE] Adding label selector for AlertManager objects discovery filtering (#2662)
+* [CHANGE] Change podManagement policy to parallel in Alertmanager (#2676)
+* [FEATURE] Provide option to turn on WAL compression (#2683)
+* [FEATURE] Support namespace denylist for listwatcher (#2710)
+* [FEATURE] Add support for Volumes to Prometheus CR (#2734)
+* [FEATURE] Add support for Volumes to Alertmanager CR (#2737)
+* [FEATURE] Add support for InitContainers to Prometheus CR (#2522)
+* [CAHNGE] Add golangci-lint and promtool to jsonnet-ci (#2739)
```
Comment (brancz) [inline CHANGELOG.md]: CAHNGE -> CHANGE

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 ## Next release
 
+## 0.32.0 / 2019-08-30
+
+* [FEATURE] Adding label selector for AlertManager objects discovery filtering (#2662)
```
Comment (brancz) [inline CHANGELOG.md]: Could you order these as described in the release document? :slightly_smiling_face:  https://github.com/coreos/prometheus-operator/blob/master/RELEASE.md#prepare-your-release

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+* [CHANGE] Change podManagement policy to parallel in Alertmanager (#2676)
+* [FEATURE] Provide option to turn on WAL compression (#2683)
+* [FEATURE] Support namespace denylist for listwatcher (#2710)
+* [FEATURE] Add support for Volumes to Prometheus CR (#2734)
```
Comment (brancz) [inline CHANGELOG.md]: Write out abbreviations.

**Comment by LiliC** [comment]
@brancz Done, PTAL

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
 ## Next release
 
+## 0.32.0 / 2019-08-30
+
+* [CHANGE] Change PodManagement policy to parallel in Alertmanager (#2676)
+* [CHANGE] Add golangci-lint and promtool to jsonnet-ci (#2739)
```
Comment (brancz) [inline CHANGELOG.md]: Only user facing changes go into the changelog, this is a tooling change. It doesn't need to be noted here.

**Comment by brancz** [comment]
Sorry one more small comment then lgtm.

File: CHANGELOG.md (hunk 4 above)
Comment (LiliC) [reply CHANGELOG.md]: Done.

**Comment by brancz** [comment]
lgtm :+1:

**Pull request merged**